### PR TITLE
[expo-face-detector] check if googleService-Info.plist exists

### PR DESCRIPTION
**Pull request by wkozyra95**

### CODE REVIEW

File: packages/expo-face-detector/ios/EXFaceDetector/EXFaceDetectorAppDelegate.m [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+#import <Firebase/Firebase.h>
+#import <UMCore/UMDefines.h>
+
+@implementation EXFaceDetectorAppDelegate
+
+UM_REGISTER_SINGLETON_MODULE(EXFaceDetectorAppDelegate)
+
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(nullable NSDictionary *)launchOptions
+{
+  NSString* googleServicesPlist = [[NSBundle mainBundle] pathForResource:@"GoogleService-Info" ofType:@"plist"];
```
Comment (sjchmiela) [inline packages/expo-face-detector/ios/EXFaceDetector/EXFaceDetectorAppDelegate.m]:
```suggestion
  NSString *googleServicesPlist = [[NSBundle mainBundle] pathForResource:@"GoogleService-Info" ofType:@"plist"];
```

File: packages/expo-face-detector/ios/EXFaceDetector/EXFaceDetectorAppDelegate.m [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  if ([FIRApp defaultApp] == nil && [[NSFileManager defaultManager] fileExistsAtPath:googleServicesPlist]) {
```
Comment (sjchmiela) [inline packages/expo-face-detector/ios/EXFaceDetector/EXFaceDetectorAppDelegate.m]:
```suggestion
  if (![FIRApp defaultApp] && [[NSFileManager defaultManager] fileExistsAtPath:googleServicesPlist]) {
```

**Review by sjchmiela — APPROVED** [review]

File: packages/expo-face-detector/ios/EXFaceDetector/EXFaceDetectorModule.m [hunk 3]
```diff
@@ -50,6 +50,10 @@ - (void)setModuleRegistry:(UMModuleRegistry *)moduleRegistry
 
 UM_EXPORT_METHOD_AS(detectFaces, detectFaces:(nonnull NSDictionary *)options resolver:(UMPromiseResolveBlock)resolve rejecter:(UMPromiseRejectBlock)reject)
 {
+  if ([FIRApp defaultApp] == nil) {
```
Comment (wkozyra95) [inline packages/expo-face-detector/ios/EXFaceDetector/EXFaceDetectorModule.m]:
```suggestion
  if (![FIRApp defaultApp]l) {
```

**Comment by rooque** [comment]
how I provide the "googleService-Info.plist" file for standalone build?

**Comment by topace124** [comment]
Hi, @sjchmiela 
Does expo-face-detector works in ejected expo project?
SDK version 34.

**Pull request merged**